packages/account/src/Sections/Security/DeactivateAccount/deactivate-account-reason.jsx

```diff
@@ -23,7 +23,8 @@ const initial_form = {
 };
 
 const preparingReason = values => {
-    let selected_reasons = selectedReasons(values)
+    let selected_reasons = '';
+    selected_reasons = selectedReasons(values)
```

Comment:
do we need this change?

Reply:
No, I was testing and forgot to revert this change 👍